infra/main.parameters.json

```diff
@@ -68,9 +80,15 @@
     "storageSkuName": {
       "value": "${AZURE_STORAGE_SKU=Standard_LRS}"
     },
+    "appServicePlanName": {
```

Comment:
For all these who have an empty string value in main.parameters.json, please just remove that empty string and put a comment of "// Set in main.parameters.json"

Reply:
Updated all the relevant variables.